src/index.ts

```diff
@@ -213,6 +220,41 @@ export class Authenticator {
     const cfDomain = request.headers.host[0].value;
     const redirectURI = `https://${cfDomain}`;
 
+    try {
+      if (this._enableLogout && request.uri === "/logout") {
+        this._logger.debug({ msg: 'Logging out', event});
+        
+        const tokens = {
+          access_token: "0",
+          refresh_token: "0",
+          id_token: this._getIdTokenFromCookie(request.headers.cookie)
+        };
+
+        const expireCookies = true;
+        const logoutUrl = `https://${this._userPoolDomain}/logout?logout_uri=${redirectURI}&client_id=${this._userPoolAppId}`;
+        return this._getRedirectResponse(tokens, cfDomain, logoutUrl, expireCookies);
```

Comment:
When cookies are cleaned up after a user logs out, the tokens contained within them are still valid. This is especially concerning for the refresh token, which is valid for 30 days by default. I would suggest at least [revoking the refresh token](https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html).